### PR TITLE
Add json suggestions for disabled capabilities

### PR DESCRIPTION
**Pull request by predragnikolic**
I never know what to put in the `disabled_capabilities`,
always have to look up the spec.

With this PR we will get JSON schema validation for `disabled_capabilities`.

### CODE REVIEW

File: sublime-package.json [hunk 1]
```diff
@@ -103,7 +103,177 @@
...
+              "markdownDescription": "A dictionary that \"masks\" (disables) the capabilities registered by this language server. For instance, to disable completions, use `{\"completionProvider\": true}`. To disable hover info, use `{\"hoverProvider\": true}`. Multiple providers can be disabled. To disable a subset of a provider, use a nested dictionary. For intance, to disable only the trigger characters of a language server, but not the completion functionality itself, use `{\"completionProvider\": {\"triggerCharacters\": true}}`.",
+              "additionalProperties": false,
```
Comment (rwols) [inline sublime-package.json:107]: Maybe allow additional properties for future protocol updates?

File: sublime-package.json [hunk 2]
```diff
@@ -103,7 +103,177 @@
...
+                "documentSymbolProvider": {
+                  "type": "boolean",
+                  "default": true
+                },
+                "codeActionProvider": {
+                  "type": "boolean",
+                  "default": true
+                },
+                "codeLensProvider": {
+                  "type": "boolean",
```
Comment (rchl) [inline sublime-package.json:184]: can you sort :)

File: sublime-package.json [hunk 3]
```diff
@@ -105,6 +105,22 @@
...
+                  "default": true
+                },
+                "colorProvider": {
+                  "type": "boolean",
+                  "default": true
+                },
```
Comment (rchl) [inline sublime-package.json:123]: I suppose all (or most, if not all) capabilities should also allow an object form, even if we don't specify the exact properties in the schema.

**Comment by rchl** [comment]
I've added a `$ref` to avoid repeating the same block.
Have not investigating whether I can merge default with non-trivial cases - thought it's not worth the effort.

**Review by rchl — APPROVED** [review]

**Pull request merged**